WebApi/Controllers/FinancialController.cs

```diff
@@ -18,27 +19,31 @@ public class FinancialController : ControllerBase
         private readonly IViewModelMapper<SalaryViewModel, Salary> _writeSalary;
         private readonly IViewModelMapper<IEnumerable<Salary>, IEnumerable<SalaryViewModel>> _readSalaryList;
         private readonly IViewModelMapper<IEnumerable<User>, IEnumerable<EmployeeViewModel>> _readEmployeesList;
+        private readonly IViewModelMapper<IEnumerable<FinancialStatement>, IEnumerable<FinancialStatementForMonthViewModel>> _finStatViewModel;
 
         public FinancialController(
             IFinancialService financialService,
             IUserService userService,
             IViewModelMapper<Salary, SalaryViewModel> readSalary,
             IViewModelMapper<SalaryViewModel, Salary> writeSalary,
             IViewModelMapper<IEnumerable<Salary>, IEnumerable<SalaryViewModel>> readSalaryList,
-            IViewModelMapper<IEnumerable<User>, IEnumerable<EmployeeViewModel>> readEmployeesList)
+            IViewModelMapper<IEnumerable<User>, IEnumerable<EmployeeViewModel>> readEmployeesList,
+            IViewModelMapper<IEnumerable<FinancialStatement>, IEnumerable<FinancialStatementForMonthViewModel>> finStatViewModel
+            )
         {
             _financialService = financialService;
             _userService = userService;
             _readSalary = readSalary;
             _writeSalary = writeSalary;
             _readSalaryList = readSalaryList;
             _readEmployeesList = readEmployeesList;
+            _finStatViewModel = finStatViewModel;
         }
 
         [HttpGet("/api/[controller]/")]
```

Comment:
Route "/api/[controller]/" for action is redundant since the controller already has this route